src/sent2vec.pyx

```diff
@@ -112,12 +113,10 @@ cdef class Sent2vecModel:
         vocab = [w.decode('utf-8', 'ignore') for w in vocab]
         freqs = list(self._thisptr.getUnigramsCounts())
         assert len(vocab) == len(freqs)
-        return {w:c for w, c in zip(vocab, freqs)}
+        return OrderedDict(zip(vocab, freqs))
 
     def get_unigram_embeddings(self):
-        vocab = list(self.get_vocabulary().items())
-        vocab.sort(key=lambda x: x[1], reverse=True)
-        vocab = [w for w, c in vocab]
+        vocab = [w for w, c in self.get_vocabulary().items()]
```

Comment:
no forcing the `<PLACEHOLDER>` token to be first? This is not modifying the order of tokens w.r.t. the previous commit. I'm fine keeping it that way, but I thought this is what you wanted to change.

Reply:
`<PLACEHOLDER>` is first in FastText by default if we don't reorder by frequency.